contrib/src/sceneeditor/collisionWindow.py

```diff
@@ -294,7 +294,7 @@ def createEntryField(self, parent, catagory, id, value,
         widget.configure(entry_state = initialState)
         widget.pack(side=LEFT)
         self.widgetDict[catagory+id] = widget
-        if defaultButton and (defaultFunction!=None):
+        if defaultButton and (defaultFunctionis notNone):
```

Comment:
It's apparently a popular style guideline to use an identity comparison instead of an equivalence comparison for the constants None, True and False.